inc/class-cachify-interface.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Interface for APC based caching storage classes.
```

Comment:
```suggestion
 * Interface for caching storage classes.
```

Well, one implementation is APC, but the others are not. The option name is just a relict of the early days where APC was the only alternative backend.